style.css

```diff
@@ -253,6 +254,90 @@ button[type="submit"] {
   font-size: 1.125rem;
 }
 
+#not-yours {
+  margin: 2rem;
+  font-size: 0.75rem;
+  text-align: center;
+}
+
+footer {
+  padding: 0 5rem 3rem 5rem;
+  background-color: #f4f4f4;
+  margin-top: 7rem;
+}
+
+h5 {
+  font-size: 1rem;
+  margin: 0.3rem 0;
+}
+
+#footer-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  margin: 0;
+}
+
+.footer-tiles {
+  padding: 3rem 0;
+  border-bottom: 1px solid rgb(228, 227, 227);
+}
+
+.footer-tiles-3 {
+  margin-top: 1.5rem;
+}
+
+footer a {
+  text-decoration: none;
+}
+
+.upper-footer-link {
+  color: #c8102e;
+}
+
+.upper-footer-link:hover {
+  text-decoration: underline;
+}
+
+.bottom-footer-link {
+  color: #572d2d;
+  display: block;
+  margin: 0.5rem 0;
+}
+
+.bottom-footer-link:hover {
+  text-decoration: underline;
+}
+
+#socials {
+  display: flex;
+  margin-left: -0.7rem;
+  gap: 1rem;
+}
+
+.social-icons {
+  display: flex;
+  height: 2rem;
+  width: 2rem;
+  justify-content: center;
+  align-items: center;
+}
+
+.social-icons a {
+  color: #572d2d;
+  margin: 1rem;
+}
+
+.social-icons:hover {
+  background-color: #c8102e;
+  color: white; 
+  /* How do we get this to work?? */
+  border-radius: 100%;
+}
+
+.social-icons a:hover {
```

Comment:
might be better with `.social-icons:hover a` so the color changes when you hover over the `div`, not just the `a`